test/IDE/comment_inherited_protocol.swift

```diff
@@ -63,23 +63,23 @@ protocol ChildProtocol : ParentProtocol1, ParentProtocol2 {
 extension ChildProtocol {
   // Should come from ParentProtocol1.
   func onlyParent1() {}
-  // CHECK: Func/onlyParent1 {{.*}} DocCommentAsXML=[<Function file="{{.*}}" line="{{.*}}" column="{{.*}}"><Name>onlyParent1()</Name><USR>s:14swift_ide_test15ParentProtocol1P11onlyParent1yyF</USR><Declaration>func onlyParent1()</Declaration><CommentParts><Abstract><Para>ParentProtocol1.onlyParent1()</Para></Abstract></CommentParts></Function>]
+  // CHECK: Func/onlyParent1 {{.*}} DocCommentAsXML=[<Function file="{{.*}}" line="{{.*}}" column="{{.*}}"><Name>onlyParent1()</Name><USR>s:14swift_ide_test13ChildProtocolPAAE11onlyParent1yyF</USR><Declaration>func onlyParent1()</Declaration><CommentParts><Abstract><Para>ParentProtocol1.onlyParent1()</Para></Abstract></CommentParts></Function>]
```

Comment:
@akyrtzi 
Previously USR used to point `ParentProtocol1.onlyParent1()` instead of `ChildProtocol.onlyParent1`.
Was that intentional? If so, I will restore that behavior.

Reply:
I'm not sure but I'd suggest to avoid changing behavior unless we know for a fact it is not intentional.